web/src/pages/index.page.tsx

```diff
@@ -47,7 +47,7 @@ const Page = () => {
         </Box>
 
         {claimDiagramId && (
-          <Box width="100%" height="100%" position="absolute">
+          <Box width="100%" height="100%" position="absolute" zIndex="2">
```

Comment:
can we add a comment why we're using 2 here? "table toolbar and pinned headers have z-index set, and we want this in front"

Reply:
sure i suppose